deploy/helm/ethereum/templates/geth-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "ethereum.fullname" . }}-geth
+  labels:
+    app: {{ template "ethereum.name" . }}
+    chart: {{ template "ethereum.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: geth
+spec:
+  replicas: {{ .Values.geth.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "ethereum.name" . }}
+      release: {{ .Release.Name }}
+      component: geth
+  template:
+    metadata:
+      labels:
+        app: {{ template "ethereum.name" . }}
+        release: {{ .Release.Name }}
+        component: geth
+    spec:
+      containers:
+      - name: geth
+        image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        command: ["/bin/sh"]
+        args: 
+        - "-c"
+        - "geth --http --http.api={{ .Values.geth.args.rpcapi }} --http.corsdomain='*' --http.addr 0.0.0.0 --http.port 8545 --networkid=3 --testnet --ethstats=${HOSTNAME}:${ETHSTATS_SECRET}@${ETHSTATS_SVC}"
+        env:
+        - name: ETHSTATS_SVC
+          value: {{ template "ethereum.fullname" . }}-ethstats.{{ .Release.Namespace }}
+        - name: ETHSTATS_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "ethereum.fullname" . }}-ethstats
+              key: WS_SECRET
```

Comment:
What is this websocket secret variable?

Reply:
That's just for the ethstats pod, nothing to do with the web socket that the relayer will use. I've not totally finished with the ETH stuff, I mostly just needed to get these doc updates merged.